payroll/views.py

```diff
@@ -103,12 +104,40 @@ def get_context_data(self, **kwargs):
         return context
 
     def median_entity_salary(self):
-        q = Salary.objects.filter(Q(job__position__employer__parent=self.object) | Q(job__position__employer=self.object))
+        q = Salary.objects.filter(
+            Q(job__position__employer__parent=self.object) |
+            Q(job__position__employer=self.object))
 
         results = q.all().aggregate(median=Percentile('amount', 0.5, output_field=FloatField()))
 
         return results['median']
 
+    def _make_pie_chart(self, container, entity_type, entity):
+        employer_payroll = entity_type.objects.filter(Q(id=entity.id)).aggregate(
```

Comment:
You only need to use `Q` objects if you have [a complex lookup](https://docs.djangoproject.com/en/3.0/topics/db/queries/#complex-lookups-with-q-objects). You can perform this lookup as a regular keyword argument to the `filter` function.